src/ic-ledger-types/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "ic-ledger-types"
+version = "0.1.0"
+edition = "2018"
+authors = ["DFINITY Stiftung <sdk@dfinity.org>"]
+description = "Types for interacting with the ICP ledger canister."
+homepage = "https://docs.rs/ic-ledger"
+documentation = "https://docs.rs/ic-ledger"
+license = "Apache-2.0"
+readme = "README.md"
+keywords = ["internet-computer", "ledger"]
+include = ["src", "Cargo.toml", "CHANGELOG.md", "LICENSE", "README.md"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ic-cdk = "0.3.3"
```

Comment:
Should we direct this dependency with path to the same repo?
```
ic-cdk = { path = "../ic-cdk", version = "0.3" }
```

Reply:
Done.

Just out of curiosity, what is the difference from the cargo point of view? As far as I can see, cargo automatically replaces this dependency with `ic-cdk = "^0.3"` on publishing.
I guess using a path dependency allows the developer to change both packages and release them in one go, but I'm not 100% sure.